scipy/stats/_kde.py

```diff
@@ -389,20 +382,12 @@ def integrate_box(self, low_bounds, high_bounds, maxpts=None):
             The result of the integral.
 
         """
-        if maxpts is not None:
-            extra_kwds = {'maxpts': maxpts}
-        else:
-            extra_kwds = {}
-
-        with MVN_LOCK:
-            value, inform = _mvn.mvnun_weighted(low_bounds, high_bounds,
-                                                self.dataset, self.weights,
-                                                self.covariance, **extra_kwds)
-        if inform:
-            msg = f'An integral in _mvn.mvnun requires more points than {self.d * 1000}'
-            warnings.warn(msg, stacklevel=2)
-
-        return value
+        low, high = low_bounds - self.dataset.T, high_bounds - self.dataset.T
```

Comment:
Confirmed that `dataset` is `atleast_2d`. Also, it looks like `low_bounds` and `high_bounds` are only supposed to work with a single point at a time, and that will always be aligned along the `-1` axis (which is why we transpose `dataset`).